.travis.yml

```diff
@@ -50,10 +50,15 @@ after_script:
       php ocular.phar code-coverage:upload --format=php-clover clover.xml
     fi
 
+stages:
+  - smoke testing
+  - test
+  - coding standard
+
 jobs:
   allow_failures:
     - php: nightly
-    - stage: Coding standard
+    - stage: coding standard
```

Comment:
The stages are lower-cased since otherwise, Travis fails to sort them properly.

Reply:
Looks like my bad. I've reverted the changes, and the order still seems correct.